### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by fd6130**
For #899 

I have update some of the docs such as remove Symfony 3 usage and retain the Guard Authenticator usage (cause this bundle still support using Guard Authenticator) along with new authenticator usage.

### CODE REVIEW

**Comment by fd6130** [comment]
Added doc regarding to decoding JWT token in a controller or service.

File: Resources/doc/index.md [hunk 1]
```diff
@@ -180,6 +194,7 @@ The following documents are available:
 - [Functionally testing a JWT protected api](3-functional-testing.md)
 - [Working with CORS requests](4-cors-requests.md)
 - [JWT encoder service customization](5-encoder-service.md)
-- [Extending JWTTokenAuthenticator](6-extending-jwt-authenticator.md)
+- [Extending Authenticator](6-extending-jwt-authenticator.md)
 - [Creating JWT tokens programmatically](7-manual-token-creation.md)
 - [A database-less user provider](8-jwt-user-provider.md)
+- [Decode JWT token](9-decode-jwt-token.md)
```
Comment (chalasr) [inline Resources/doc/index.md]: I fear this might be slightly confusing. What about `Accessing the authenticated JWT token`?

Comment (fd6130) [reply Resources/doc/index.md]: Great, i will change it.

**Review by chalasr — APPROVED** [review]
Awesome!

**Comment by chalasr** [comment]
Thanks a lot, @fd6130.

**Pull request merged**